tests/simulate/test_dilute.py

```diff
@@ -45,7 +45,7 @@ def test_run(self):
         d = relentless.simulate.Dilute(init, operations=md)
         sim = d.run(potentials=pots, directory=self.directory)
         ens_ = sim[analyzer]["ensemble"]
-        self.assertAlmostEqual(ens_.P, -207.5228556)
+        self.assertAlmostEqual(ens_.P, 194.8990634)
```

Comment:
I think these might be due to the errors above. The numbers might be a little different but shouldn't have changed sign completely, I don't think... Let me know if that's not the case.

Also, we should add a test for MD with a barostat.

Reply:
I had missed a negative sign in front of b_ij!